File: 프로그래머스/1/12948. 핸드폰 번호 가리기/핸드폰 번호 가리기.java
class Solution {
    public String solution(String phone_number) {
        String answer = "";
        String a = "";
        for (int i = 0; i < phone_number.length()-4; i++) {
            a += "*";
        }
        String b = phone_number.substring(phone_number.length()-4,phone_number.length());
        answer = a+b;
        return answer;
    }
}

File: 프로그래머스/1/12948. 핸드폰 번호 가리기/핸드폰 번호 가리기.swift
func solution(_ phone_number:String) -> String {
    var last = phone_number.suffix(4)
    var a = phone_number.count - 4
    var ans = ""
    if a > 0 {
        for i in 0..<a{
            ans += "*"
        }
    }
    ans += last
    
    return ans
}
